docs/src/components/home/Hero.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import ContentCopyRounded from '@material-ui/icons/ContentCopyRounded';
+import GradientText from 'docs/src/components/GradientText';
+
+const Hero = () => {
+  return (
+    <Box sx={{ overflow: 'hidden' }}>
+      <Container sx={{ minHeight: 500, height: 'calc(100vh - 120px)', maxHeight: 700 }}>
+        <Grid
+          container
+          alignItems="center"
+          wrap="nowrap"
+          sx={{ height: '100%', maxWidth: { xs: 500, md: 'initial' }, mx: 'auto' }}
+        >
+          <Grid item md={7} lg={6}>
+            <Box textAlign={{ xs: 'center', md: 'left' }} maxWidth={500}>
+              <Typography
+                variant="h1"
+                fontWeight={800}
+                fontSize="clamp(2.625rem, 1.2857rem + 3.5714vw, 4.5rem)"
+                lineHeight={1.11}
+                mb={2}
+              >
+                The <GradientText>ultimate</GradientText> solution for your UI.
+              </Typography>
+              <Typography variant="body1" color="text.secondary" mb={3}>
+                MUI provides robust, customizable, and accessible components, enabling you to build
+                React applications faster. With a trusted open-source community and beautiful
+                designs, developing UIs have never been easier.
+              </Typography>
+              <Box
+                display="flex"
+                flexWrap="wrap"
+                justifyContent={{ xs: 'center', md: 'flex-start' }}
+                sx={{ '&& > *': { minWidth: 'clamp(0px, (449px - 100%) * 999 ,100%)' } }}
+              >
+                <Button size="large" variant="contained" endIcon={<KeyboardArrowRightRounded />}>
+                  Get Started
+                </Button>
+                <Box width={16} height={16} />
+                <Button
+                  size="large"
+                  variant="outlined"
+                  endIcon={<ContentCopyRounded sx={{ color: 'grey.500' }} />}
+                  sx={{
+                    color: 'grey.600',
+                    border: '1px solid #e5e8ec',
+                    bgcolor: '#f3f6f9',
+                  }}
+                >
+                  <Box component="span" sx={{ mr: 1, color: 'grey.400' }}>
+                    $
+                  </Box>{' '}
+                  npm install @mui/core
+                </Button>
+              </Box>
+            </Box>
+          </Grid>
+          <Grid
```

Comment:
Little detail I put here is a `border-bottom-left-radius` of 10px :D

Reply:
👍